controllers/storagecluster/cephcluster_test.go

```diff
@@ -1193,3 +1195,71 @@ func TestCephClusterNetworkConnectionsSpec(t *testing.T) {
 		assert.DeepEqual(t, cc.Spec.Network.Connections, testCase.ccSpec.Network.Connections)
 	}
 }
+
+func TestGetIPFamilyConfig(t *testing.T) {
```

Comment:
@sp98 The unit tests are still failing PTAL

Reply:
should be ok now.